frameworks/cassandra/tests/test_overlay.py

```diff
@@ -22,25 +22,27 @@
 OVERLAY_OPTIONS = {'service':{'virtual_network':True}}
 
 
-def setup_module(module):
-    install.uninstall(PACKAGE_NAME)
-    utils.gc_frameworks()
-
-    # check_suppression=False due to https://jira.mesosphere.com/browse/CASSANDRA-568
-    install.install(PACKAGE_NAME, DEFAULT_TASK_COUNT, check_suppression=False,
-                    additional_options=OVERLAY_OPTIONS)
-    plan.wait_for_completed_deployment(PACKAGE_NAME)
-    tmp_dir = tempfile.mkdtemp(prefix='cassandra-test')
-    for job in TEST_JOBS:
-        jobs.install_job(job, tmp_dir=tmp_dir)
-
-
-def teardown_module(module):
-    install.uninstall(PACKAGE_NAME)
-
-    for job in TEST_JOBS:
-        jobs.remove_job(job)
-
+@pytest.fixture(scope='module', autouse=True)
+def configure_package():
```

Comment:
Since we are making this a fixture, can we make this one not autouse, and just have it explicitly use `configure_universe`? E.G.
```
@pytest.fixture(scope='module')
def configure_package(configure_universe):
    ...
```
This is technically the exact same thing as the autouse, but its a better precedent as autouse can easily get out of control